pavelib/utils/test/suites/bokchoy_suite.py

```diff
@@ -30,6 +35,9 @@ class BokChoyTestSuite(TestSuite):
       testsonly - assume servers are running (as per above) and run tests with no setup or cleaning of environment
       test_spec - when set, specifies test files, classes, cases, etc. See platform doc.
       default_store - modulestore to use when running tests (split or draft)
+      num_processes - number of processes or threads to use in tests. Recommendation is that this
+      is equal to the number of available processors.
```

Comment:
Nit: we've been using 2 processes on 4-core machines, maybe this docstring should say "at most" instead of "is equal" to match this common usage?